ios/AppLovinMAXNativeAdView.m

```diff
@@ -318,6 +318,8 @@ - (void)sendAdLoadedReactNativeEventForAd:(MANativeAd *)ad
             jsNativeAd[@"image"] = @(YES);
         }
     }
+    jsNativeAd[@"optionsView"] = ad.optionsView ? @(YES) : @(NO);
+    jsNativeAd[@"mediaView"] = ad.mediaView ? @(YES) : @(NO);
```

Comment:
How is `"optionsView"` and `"mediaView"` related to this PR?

Reply:
These new flags will be used to check the native component is available from the ad network in the JavaScript components.   This is the same as checking `nativeAd.title` and `nativeAd.advertiser`.

This can prevent the JavaScript component from accessing the native platform side when the native component is unavailable.

Reply:
- Where are we using them though, don't see it in demo app either
- Since these are booleans, you should name the flags as such. `isOptionsViewAvailable` and `isMediaViewAvailable`.

Reply:
But again, this PR is for updating init logic, and these flags seem unrelated to this PR

Reply:
These flags will be used only in `src/NativeAdComponents.js`.   The values defined in `jsNativeAd` are not intended for the users to see. 

Right, this is piggybacking on the init logic PR.  Let me separate these NativeAdComponents changes from this PR.  This will be more clear.